customers.go

```diff
@@ -36,6 +36,7 @@ type Customer struct {
 	Zip                string `json:"zip,omitempty"`
 	LeadCreatedAt      string `json:"lead_created_at,omitempty"`
 	FreeTrialStartedAt string `json:"free_trial_started_at,omitempty"`
+	WebsiteUrl 		   string `json:"website_url,omitempty"`
```

Comment:
[minor] It would be great to match code indentation of `WebsiteUrl`

```
FreeTrialStartedAt string `json:"free_trial_started_at,omitempty"`
WebsiteUrl         string `json:"website_url,omitempty"`
```